ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/baselinecaps/none/provision/cucushift-installer-rehearse-aws-ipi-baselinecaps-none-provision-chain.yaml

```diff
@@ -0,0 +1,17 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-baselinecaps-none-provision
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
```

Comment:
if it is `baselinecaps-non`, I guess we do need to enable `enable-qe-catalogsource` any more. cc @jianzhangbjz for double confirm

Reply:
Actually, we need to add it. The `enable-qe-catalogsource` will provide a catalog source for all optional operator teams to run their test cases. Even if the `marketplace` disable, the CatalogSource will still work. But, it will use the `openshift-marketplace` project. So, this step will fail now. I will update the `enable-qe-catalogsource` step to adapt to this cluster profile.

Reply:
thanks for the confirm, I removed  enable-qe-catalogsource, will add back when the fix in enable-qe-catalogsource is available.